tests/system/action/user/test_create.py

```diff
@@ -290,68 +346,236 @@ def test_create_permission_superadmin(self) -> None:
             },
         )
 
```

Comment:
Please add a test with `"organisation_management_level": null,` in the payload and see, that it works (maybe in the group D section)

Reply:
done in test_create.py: test_create_permission_group_D_permission_with_OML